packages/backend/src/config.js

```diff
@@ -53,6 +54,17 @@ config.max_subdomains_per_user = 2000;
 config.storage_capacity = 1*1024*1024*1024;
 config.static_hosting_domain = '-static.puter.local';
 
+// Storage limiting is set to false by default
+// Storage available on the device puter is running is the storage available
+config.is_storage_limited = false;
+diskusage.check('/', (err, info) => {
```

Comment:
This could be dynamic if it's moved to a service. I'm going to merge as-is for now though since this is better than the current state of things.